modules/tts/tts-adapters/tts-adapter-sapinative/sapinative.sln

```diff
@@ -1,9 +1,16 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.32106.194
+# Visual Studio Version 17
```

Comment:
You updated to Visual Studio 17 and in the commit message you say:

> updated the projects to use Visual Studio 2022 and updated tool chain

but the README says "Visual studio 2019 or newer". Is that still correct or does it need updating?

Reply:
(sorry i was on the move for AVH yesterday)
You are right i need to change this to Visual studio 2022
For the version in the sln file, i think 2022 is the version 17 of visual studio.

i'll do a pass on the code this afternoon.

Reply:
@NPavie do you want to update the README?

Reply:
@NPavie This PR is almost ready to merge, only this small thing left.

Reply:
Great. Thanks!